src/foregroundTimeTracker.ts

```diff
@@ -0,0 +1,72 @@
+export default class ForegroundTimeTracker {
```

Comment:
Why does it reset to 0 when I open a NEW tab.  Behavior after opening and switching back and forth seems to work though.

Reply:
resolved!